.changeset/lazy-beds-visit.md

```diff
@@ -0,0 +1,5 @@
+---
+'@spreadshirt/backstage-plugin-s3-viewer-backend': patch
+---
+
+Add missing `region` parameter in README file
```

Comment:
I don't think this needs a changeset, as it is not a code change?

Reply:
I wasn't really sure either. I will just remove it then